cpp/include/cudf/functions.h

```diff
@@ -758,23 +758,21 @@ gdf_error gdf_hash_partition(int num_input_cols,
                              int partition_offsets[],
                              gdf_hash_func hash);
 
-/* prefixsum */
+/* prefix scan */
 /** --------------------------------------------------------------------------*
- * @brief  Computes the prefix sum of a column.
+ * @brief  Computes the prefix scan of a column.
```

Comment:
In this PL, the supported operators of `gdf_scan` are `sum`, `min`, `max`, `product`.
I'd like to change "the prefix scan" into "the scan".

Reply:
That works. I suggest "Computes the scan (aka prefix sum) of a column", because some people may know the term "prefix sum" and not "scan". The term "prefix sum" is sometimes used even when the operator is not +.

Reply:
Okay, I will update it.
Since I'm working on cython update, the document update will be after cython update.
I'm also looking PR #577